src/qtwebkit.mk

```diff
@@ -4,11 +4,11 @@ PKG             := qtwebkit
 $(PKG)_WEBSITE  := https://github.com/annulen/webkit
 $(PKG)_DESCR    := QtWebKit
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 5.212
-$(PKG)_CHECKSUM := fada6fa2f2bae27a16b666e0360680de6ab21aac044670b434a729334e57b154
+$(PKG)_VERSION  := 5.13.1
+$(PKG)_CHECKSUM := de062c94dcc163b935f4f957a08f5b32cfcb5f15cdfb67ce8a036ce4f222edb3
 $(PKG)_SUBDIR   := qtwebkit-everywhere-src-$($(PKG)_VERSION)
 $(PKG)_FILE     := qtwebkit-everywhere-src-$($(PKG)_VERSION).tar.xz
-$(PKG)_URL      := https://download.qt.io/snapshots/ci/qtwebkit/$($(PKG)_VERSION)/latest/src/submodules/$($(PKG)_FILE)
+$(PKG)_URL      := https://download.qt.io/snapshots/ci/qtwebkit/5.212/latest/src/submodules/$($(PKG)_FILE)
```

Comment:
Why would the file not be in the versioned path?

Reply:
Ask Qt developers why they have put QtWebkit 5.13.1 into the path containing 5.212...